jobs/jac.groovy

```diff
@@ -0,0 +1,32 @@
+job('generate docs') {
+    scm {
+        git {
+            remote {
+                name('origin')
+                url("https://github.com/cfpb/jenkins-automation")
+            }
+        }
+    }
+
+    steps{
+        gradle {
+            tasks('groovydoc')
+            useWrapper()
+        }
+        shell("""
+            git add -A
+            git commit -m'Jenkins autogenerate docs'
+       """)
+    }
+    triggers {
+        scm 'H/5 * * * *'
+    }
+    publishers {
+        git {
+            pushOnlyIfSuccess()
+            pushMerge(false)
+            forcePush(true)
```

Comment:
It force pushes the docs to gh-pages, to avoid code conflicts it will overwrite the docs to the latest everytime.